back-end/Service.py

```diff
@@ -0,0 +1,280 @@
+    return jsonify(or_handler.delete_order(data))
+             
+@app.route('/deleteLocationType', methods=['POST'])
+def delete_location_type():
+    data = json.loads(request.stream.read())
+    db = DataAccessLocation()
+    loc_handler = LocationHandler(db)
+    return jsonify(loc_handler.delete_location_type(data))
+
+
+@app.route('/updateOrder', methods=['POST'])
+def update_order():
+    data = json.loads(request.stream.read())
+    db = DataAccessOrders()
+    or_handler = OrderHandler(db)
+    order = Order( data["date"], data["time"],data["name"],data["telefon"],data["email"],data["location"], data["eventType"],data["note"], data["id"])
+    return jsonify(or_handler.update_order(order))
+
+@app.route('/getTimeSlots') 
+def get_all_time_slots():
+    db = DataAccessCalendar()
+    tu_handler = TimeUnitHandler(db)
+    return jsonify(tu_handler.get_time_slots())
+
+
+@app.route('/getWeeklyOpenSlots')
+def get_weekly_slots():
+    db = DataAccessCalendar()
+    tu_handler = TimeUnitHandler(db)
+    return jsonify(tu_handler.get_weekly_time_slots())
+
+@app.route('/getSingleOpenSlots')
+def get_single_slots():
+    db = DataAccessCalendar()
+    tu_handler = TimeUnitHandler(db)
+    return jsonify(tu_handler.get_single_time_slots())
+
+
+@app.route('/addorder', methods=['POST']) 
+def add_order():
+    data = json.loads(request.stream.read())
+    db = DataAccessOrders()
+    or_handler = OrderHandler(db)
+    order = Order( data["date"], data["time"],data["name"],data["telefon"],data["email"],data["location"], data["eventType"],data["note"])
+    mail = Mail()
+    mail.send_mail(order)
+    return jsonify(or_handler.add_new_order(order))
+
+@app.route('/getOrders')
+def get_orders():
+    db = DataAccessOrders()
+    or_handler = OrderHandler(db)
+    return jsonify(or_handler.get_orders())
+
+@app.route('/getLocationsInfo')
+def get_locations_info():
+    db = DataAccessLocation()
+    loc_info_handler = LocationHandler(db)
+    return jsonify(loc_info_handler.get_all_locations_info())
+
+@app.route('/getLocations')
+def get_locations():
+    db = DataAccess()
+    loc_info_handler = LocationHandler(db)
+    return jsonify(loc_info_handler.get_all_locations())
+
+
+
+
+@app.route('/getLocationTypes')
+def get_location_types():
+    db = DataAccess()
+    loc_info_handler = LocationHandler(db)
+    return jsonify(loc_info_handler.get_all_locations_types())
+
+@app.route('/addLocation', methods=['POST'])
+def add_location():
+    data = json.loads(request.stream.read())
+    location = Location(data["name"],data["type"],data["latitude"],data["longitude"], data["description"] )
+    loc_DB = DataAccessLocation()
+    loc_handler = LocationHandler(loc_DB)
+    loc_handler.add_location(location)
+    print(data)
+    image = Image(data["sphereImageList"][0]['name'], "sphere", data["type"], data["name"])
+    im_DB = DataAccessImage()
+    im_handler = ImageHandler(im_DB)
+    im_handler.add_image(image)
+    return '200'

```

Comment:
this should return "accepted" or something

Reply:
Do you have some related article about this issue?